Jenkinsfile

```diff
@@ -155,6 +155,8 @@ def RunIntegrationTest(k, v) {
                     }
                     finally{
                         sh "journalctl -u kubelet > _output/tests/kubelet_${v}.log"
```

Comment:
should the kubelet log also use `_${k}_${v}` in its file name for consistency?

Reply:
I'd suggest we keep the names simpler with just `_${v}`. Or at least I don't see why we need both. See [these artifacts](https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/PR-2097/2/artifacts) for an example.

Reply:
Would we ever have a scenario where two minor versions will be supported in our CI? eg: 1.11.1 and 1.11.2? if not, i can change the patch to just hold `${v}` for all the files.

Reply:
It’s possible that we would have two different minor versions, but the ${v} should still ensure it is unique

Reply:
That is very true. Alright if i update the patch removing ${k} from all the files?

Reply:
Yes sounds good. Since we haven’t been able to repro the issue yet for troubleshooting before merge, we can go ahead and merge after that.

Reply:
@ganeshmaharaj would you mind squashing the commits? Thanks

Reply:
@travisn done and also rebased the patch to master.